src/Core/Grid/Presenter/GridPresenter.php

```diff
@@ -55,7 +55,7 @@ public function present(GridInterface $grid)
         $searchCriteria = $grid->getSearchCriteria();
         $data = $grid->getData();
         $presentedGrid = [
-            'id' => $definition->getId(),
+            'id' => strtolower($definition->getId()),
```

Comment:
i dont like this workaround, but its related to this issue https://github.com/PrestaShop/PrestaShop/issues/10412 :(